lib/src/solver/report.dart

```diff
@@ -143,6 +143,23 @@ class SolveReport {
     }
   }
 
+  /// Displays a single-line message, number of discontinued packages
+  /// if discontinued packages are detected.
+  Future<void> reportDiscontinued() async {
+    var numDiscontinued = 0;
```

Comment:
Minor nit: follow the style from reportOutdated.
```
final discontinuedCount = result.packages.where((pkg) => ...).length;
```

Reply:
Yes, I tried to do it using `where` but the method needs to async which is not possible with it.

Reply:
Yes, I tried to do it using `where` but the method needs to async which is not possible with it.

Reply:
Ah! - good point. Did not think of that. Then your solution is  fine.